### ISSUE TITLE
Reference hints for managed dlls on EA Origin

### ISSUE THREAD

**Issue by aubergine10** [CitiesSkylinesMods/TMPE#760]
Addition of reference path hints for when developer has installed game via EA Origin game portal - the managed .dll files are in different location.

Full details see:  #751

This registry key in particular looks useful and has been confirmed to exist:

```reg
[HKEY_LOCAL_MACHINE\SOFTWARE\Electronic Arts\EA Core]
"EADM6InstallDir"="C:\\Program Files\\Origin"
```

^ Folder above just an example. Seems to be `Program Files (x86)\Origin Games` in recent versions.

Default managed dll folder on Origin is:

```
C:\Program Files (x86)\Origin Games\Cities Skylines\Cities_Data\Managed
```

The zip in https://github.com/CitiesSkylinesMods/TMPE/pull/751#issuecomment-592790424 lists all registry keys mentioning `Cities.exe` and `Origin`.

**Comment by kianzarrin** [comment]
is this fixed

**Comment by krzychu124** [comment]
I'm pretty sure we've merged all changes so no idea if something else needs to be done. I would close it _(I've never heard of anyone using origin build)_.

**Comment by aubergine10** [comment]
Yes, it's fixed by #751 - I'll update changelogs and close this shortly.
